<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>DEGs</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">invitro</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li>
  <a href="degs.html">DEGs</a>
</li>
<li>
  <a href="common_genes.html">Common genes</a>
</li>
<li>
  <a href="unique_genes.html">Unique genes</a>
</li>
<li>
  <a href="single_cell.html">Single cell</a>
</li>
<li>
  <a href="cell_cycle.html">Cell cycle</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdhenaos/invitro">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">DEGs</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-08-28
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>invitro/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.1). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250827code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20250827)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250827code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250827)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/workspaces/invitro/invitro/data/
</td>
<td style="text-align:left;">
data
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomjdhenaosinvitrotree39f50392e3e8ae70eb6323c5c34137738eaba12dtargetblank39f5039a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/jdhenaos/invitro/tree/39f50392e3e8ae70eb6323c5c34137738eaba12d" target="_blank">39f5039</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomjdhenaosinvitrotree39f50392e3e8ae70eb6323c5c34137738eaba12dtargetblank39f5039a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/jdhenaos/invitro/tree/39f50392e3e8ae70eb6323c5c34137738eaba12d" target="_blank">39f5039</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Untracked files:
    Untracked:  code/cell_cycle.r
    Untracked:  code/common_genes.r
    Untracked:  code/deg_plots.r
    Untracked:  code/single_cell.r
    Untracked:  code/unique_genes.r
    Untracked:  data/220427_ATII_embedding.rds
    Untracked:  data/220427_EC_embedding.rds
    Untracked:  data/220427_MFB_embedding.rds
    Untracked:  data/G1S.txt
    Untracked:  data/G2M.txt
    Untracked:  data/Hyperoxia_AT.xls
    Untracked:  data/Hyperoxia_EC.xls
    Untracked:  data/Hyperoxia_MFB.xls
    Untracked:  data/bulk_seurat_endothelial.rds
    Untracked:  data/bulk_seurat_epithelial.rds
    Untracked:  data/bulk_seurat_stroma.rds
    Untracked:  data/inVivo_hyperoxia.xlsx
    Untracked:  et(frame,nsets = 20, nintersects = 15,

Unstaged changes:
    Modified:   README.md
    Modified:   _workflowr.yml
    Modified:   data/README.md
    Modified:   invitro.Rproj

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/degs.Rmd</code>) and HTML (<code>docs/degs.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jdhenaos/invitro/blob/39f50392e3e8ae70eb6323c5c34137738eaba12d/analysis/degs.Rmd" target="_blank">39f5039</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-28
</td>
<td>
workflowr::wflow_publish("analysis/*")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jdhenaos/invitro/5d60f00e8080c600f25738917a124a2550b243b7/docs/degs.html" target="_blank">5d60f00</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jdhenaos/invitro/bb42f18cc2a7f5be1811be1890a0567220efc35d/docs/degs.html" target="_blank">bb42f18</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jdhenaos/invitro/blob/31a62a2a8a65822fa868a61c698412ce9d433212/analysis/degs.Rmd" target="_blank">31a62a2</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
<td>
Add DEGs analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jdhenaos/invitro/7603db9880bd72cb6f605aaf20cb7e600cae662c/docs/degs.html" target="_blank">7603db9</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jdhenaos/invitro/blob/d65917bc5b9d473c2477ce3f23a5c36125e7c905/analysis/degs.Rmd" target="_blank">d65917b</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
<td>
Start my new project
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(ggplot2)
library(UpSetR)
library(ggrepel)

data.folder &lt;- &quot;/workspaces/invitro/invitro/data/&quot;

raw.AT &lt;- read.table(paste0(data.folder,&quot;Hyperoxia_AT.xls&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)
raw.EC &lt;- read.table(paste0(data.folder,&quot;Hyperoxia_EC.xls&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)
raw.MFB &lt;- read.table(paste0(data.folder,&quot;Hyperoxia_MFB.xls&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)

DEG.AT &lt;- raw.AT[which(raw.AT$padj &lt; 0.05),]
DEG.EC &lt;- raw.EC[which(raw.EC$padj &lt; 0.05),]
DEG.MFB &lt;- raw.MFB[which(raw.MFB$padj &lt; 0.05),]</code></pre>
<div id="upset-plot" class="section level1">
<h1>UpSet plot</h1>
<pre class="r"><code>frame &lt;- fromList(list(DEG.EC$ensembl_gene_id,
                       DEG.MFB$ensembl_gene_id,
                       DEG.AT$ensembl_gene_id))

colnames(frame) &lt;- c(paste0(&quot;EC Hyperoxia&quot;,&quot; (N = &quot;,nrow(DEG.EC),&quot;)&quot;),
                     paste0(&quot;FB Hyperoxia&quot;,&quot; (N = &quot;,nrow(DEG.MFB),&quot;)&quot;),
                     paste0(&quot;AT Hyperoxia&quot;,&quot; (N = &quot;,nrow(DEG.AT),&quot;)&quot;))

upset(frame,nsets = 20, nintersects = 15,
      queries = list(list(query = intersects, 
                          params = list(&quot;AT Hyperoxia (N = 1368)&quot;,&quot;FB Hyperoxia (N = 1150)&quot;), 
                          color = &quot;#C6DBEF&quot;, active = T),
                     list(query = intersects, 
                          params = list(&quot;AT Hyperoxia (N = 1368)&quot;,&quot;FB Hyperoxia (N = 1150)&quot;,&quot;EC Hyperoxia (N = 69)&quot;), 
                          color = &quot;#4292C6&quot;, active = T)),
      order.by = c(&quot;freq&quot;),empty.intersections = NULL,
      sets.bar.color = c(&quot;#A79AD4&quot;,&quot;#DF758C&quot;,&quot;#ACD49F&quot;),
      sets = colnames(frame),keep.order = TRUE)</code></pre>
<p><img src="figure/degs.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jdhenaos/invitro/blob/7603db9880bd72cb6f605aaf20cb7e600cae662c/docs/figure/degs.Rmd/unnamed-chunk-2-1.png" target="_blank">7603db9</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="volcano-plot" class="section level1">
<h1>Volcano plot</h1>
<pre class="r"><code>raw.AT &lt;- raw.AT[order(raw.AT$padj),1:8]
raw.EC &lt;- raw.EC[order(raw.EC$padj),1:8]
raw.MFB &lt;- raw.MFB[order(raw.MFB$padj),1:8]

raw.AT$direction &lt;- &quot;No-significant&quot;
raw.EC$direction &lt;- &quot;No-significant&quot;
raw.MFB$direction &lt;- &quot;No-significant&quot;

raw.AT$direction[which(raw.AT$padj &lt; 0.05 &amp; raw.AT$log2FoldChange &gt; 0)] &lt;- &quot;AT Up-regulated&quot;
raw.EC$direction[which(raw.EC$padj &lt; 0.05 &amp; raw.EC$log2FoldChange &gt; 0)] &lt;- &quot;EC Up-regulated&quot;
raw.MFB$direction[which(raw.MFB$padj &lt; 0.05 &amp; raw.MFB$log2FoldChange &gt; 0)] &lt;- &quot;FB Up-regulated&quot;

raw.AT$direction[which(raw.AT$padj &lt; 0.05 &amp; raw.AT$log2FoldChange &lt; 0)] &lt;- &quot;AT Down-regulated&quot;
raw.EC$direction[which(raw.EC$padj &lt; 0.05 &amp; raw.EC$log2FoldChange &lt; 0)] &lt;- &quot;EC Down-regulated&quot;
raw.MFB$direction[which(raw.MFB$padj &lt; 0.05 &amp; raw.MFB$log2FoldChange &lt; 0)] &lt;- &quot;FB Down-regulated&quot;

raw.AT$threshold &lt;- c(rep(TRUE,5),rep(FALSE,nrow(raw.AT) - 5))
raw.EC$threshold &lt;- c(rep(TRUE,5),rep(FALSE,nrow(raw.EC) - 5))
raw.MFB$threshold &lt;- c(rep(TRUE,5),rep(FALSE,nrow(raw.MFB) - 5))

all.hyperoxia &lt;- rbind(raw.AT,raw.MFB,raw.EC)
all.hyperoxia$direction &lt;- as.factor(all.hyperoxia$direction)
head(all.hyperoxia[order(all.hyperoxia$padj),],10)</code></pre>
<pre><code>          ensembl_gene_id mgi_symbol  baseMean log2FoldChange      lfcSE
19009  ENSMUSG00000023015    Racgap1 2126.7225     -1.1347772 0.09047989
21849  ENSMUSG00000025001      Hells 1218.9385     -0.9741433 0.08294554
188171 ENSMUSG00000022360      Atad2 2370.5353     -0.8715216 0.07697328
764    ENSMUSG00000026355       Mcm6 4053.6186     -0.6640666 0.06029915
2646   ENSMUSG00000027469       Tpx2 2590.7041     -1.0022389 0.09216284
13727  ENSMUSG00000036777       Anln 6570.4818     -0.6903815 0.06344467
14488  ENSMUSG00000032555     Topbp1 4023.1976     -0.5929167 0.05434847
13113  ENSMUSG00000037544     Dlgap5  909.1766     -1.1617465 0.10910280
160101 ENSMUSG00000069793      Slfn9  988.9436     -1.3188793 0.12277972
76910  ENSMUSG00000026355       Mcm6 3211.9482     -0.7980071 0.07483324
            stat       pvalue         padj         direction threshold
19009  -12.54176 4.410857e-36 5.775577e-32 AT Down-regulated      TRUE
21849  -11.74437 7.547978e-32 4.941661e-28 AT Down-regulated      TRUE
188171 -11.32239 1.016601e-29 1.309281e-25 FB Down-regulated      TRUE
764    -11.01287 3.312903e-28 1.445972e-24 AT Down-regulated      TRUE
2646   -10.87465 1.522340e-27 3.322254e-24 AT Down-regulated      TRUE
13727  -10.88163 1.410184e-27 3.322254e-24 AT Down-regulated      TRUE
14488  -10.90954 1.037828e-27 3.322254e-24 AT Down-regulated     FALSE
13113  -10.64818 1.778033e-26 3.325938e-23 AT Down-regulated     FALSE
160101 -10.74183 6.474572e-27 4.169301e-23 FB Down-regulated      TRUE
76910  -10.66381 1.503173e-26 6.453123e-23 FB Down-regulated      TRUE</code></pre>
<pre class="r"><code>result &lt;- all.hyperoxia[which(!is.na(all.hyperoxia$padj)),]
result &lt;- result[order(result$threshold,decreasing = TRUE),]

ggplot(result) +
  geom_point(aes(x=log2FoldChange, y=-log10(padj),colour=direction)) + 
  scale_color_manual(values=c(&quot;#ACD49F&quot;,&quot;#008F00&quot;,&quot;#A79AD4&quot;,&quot;#7030A0&quot;,&quot;#DF758C&quot;,&quot;#E7298A&quot;,&quot;grey&quot;)) +
  geom_text_repel(data =subset(result[1:15,], threshold == TRUE), 
                  mapping =aes(x= log2FoldChange, y = -log10(padj), label = result$mgi_symbol[1:15]), size = 3) +
  xlab(&quot;log2 fold change&quot;) + 
  ylab(&quot;-log10 adjusted p-value&quot;) +
  geom_hline(yintercept=-log10(0.05), linetype=&quot;dashed&quot;, color = &quot;red&quot;) + theme_classic() + theme(legend.title = element_blank())</code></pre>
<p><img src="figure/degs.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jdhenaos/invitro/blob/7603db9880bd72cb6f605aaf20cb7e600cae662c/docs/figure/degs.Rmd/unnamed-chunk-3-1.png" target="_blank">7603db9</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-08-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="session-info" class="section level1">
<h1>Session info</h1>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-pc-linux-gnu
Running under: Ubuntu 22.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Etc/UTC
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggrepel_0.9.6   UpSetR_1.4.0    ggplot2_3.5.2   workflowr_1.7.1

loaded via a namespace (and not attached):
 [1] gtable_0.3.6       jsonlite_2.0.0     dplyr_1.1.4        compiler_4.4.0    
 [5] promises_1.3.3     tidyselect_1.2.1   Rcpp_1.1.0         stringr_1.5.1     
 [9] git2r_0.33.0       gridExtra_2.3      callr_3.7.6        later_1.4.4       
[13] jquerylib_0.1.4    scales_1.4.0       yaml_2.3.10        fastmap_1.2.0     
[17] plyr_1.8.9         R6_2.6.1           labeling_0.4.3     generics_0.1.4    
[21] knitr_1.50         tibble_3.3.0       rprojroot_2.1.1    RColorBrewer_1.1-3
[25] bslib_0.9.0        pillar_1.11.0      rlang_1.1.6        cachem_1.1.0      
[29] stringi_1.8.7      httpuv_1.6.16      xfun_0.53          getPass_0.2-4     
[33] fs_1.6.6           sass_0.4.10        cli_3.6.5          withr_3.0.2       
[37] magrittr_2.0.3     ps_1.9.1           digest_0.6.37      grid_4.4.0        
[41] processx_3.8.6     rstudioapi_0.17.1  lifecycle_1.0.4    vctrs_0.6.5       
[45] evaluate_1.0.4     glue_1.8.0         farver_2.1.2       whisker_0.4.1     
[49] rmarkdown_2.29     httr_1.4.7         tools_4.4.0        pkgconfig_2.0.3   
[53] htmltools_0.5.8.1 </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-pc-linux-gnu
Running under: Ubuntu 22.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Etc/UTC
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggrepel_0.9.6   UpSetR_1.4.0    ggplot2_3.5.2   workflowr_1.7.1

loaded via a namespace (and not attached):
 [1] gtable_0.3.6       jsonlite_2.0.0     dplyr_1.1.4        compiler_4.4.0    
 [5] promises_1.3.3     tidyselect_1.2.1   Rcpp_1.1.0         stringr_1.5.1     
 [9] git2r_0.33.0       gridExtra_2.3      callr_3.7.6        later_1.4.4       
[13] jquerylib_0.1.4    scales_1.4.0       yaml_2.3.10        fastmap_1.2.0     
[17] plyr_1.8.9         R6_2.6.1           labeling_0.4.3     generics_0.1.4    
[21] knitr_1.50         tibble_3.3.0       rprojroot_2.1.1    RColorBrewer_1.1-3
[25] bslib_0.9.0        pillar_1.11.0      rlang_1.1.6        cachem_1.1.0      
[29] stringi_1.8.7      httpuv_1.6.16      xfun_0.53          getPass_0.2-4     
[33] fs_1.6.6           sass_0.4.10        cli_3.6.5          withr_3.0.2       
[37] magrittr_2.0.3     ps_1.9.1           digest_0.6.37      grid_4.4.0        
[41] processx_3.8.6     rstudioapi_0.17.1  lifecycle_1.0.4    vctrs_0.6.5       
[45] evaluate_1.0.4     glue_1.8.0         farver_2.1.2       whisker_0.4.1     
[49] rmarkdown_2.29     httr_1.4.7         tools_4.4.0        pkgconfig_2.0.3   
[53] htmltools_0.5.8.1 </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
