<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>in vivo</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">invitro</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li>
  <a href="degs.html">DEGs</a>
</li>
<li>
  <a href="common_genes.html">Common genes</a>
</li>
<li>
  <a href="unique_genes.html">Unique genes</a>
</li>
<li>
  <a href="single_cell.html">Single cell</a>
</li>
<li>
  <a href="cell_cycle.html">Cell cycle</a>
</li>
<li>
  <a href="at1_at2.html">AT1 vs AT2</a>
</li>
<li>
  <a href="invivo.html">in vivo</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdhenaos/invitro">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">in vivo</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-09-02
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>invitro/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.1). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250827code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20250827)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250827code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250827)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/workspaces/invitro/invitro/data/
</td>
<td style="text-align:left;">
data
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomjdhenaosinvitrotree2bba1591ff879590b7dc34d96e8a49ec2433c050targetblank2bba159a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/jdhenaos/invitro/tree/2bba1591ff879590b7dc34d96e8a49ec2433c050" target="_blank">2bba159</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomjdhenaosinvitrotree2bba1591ff879590b7dc34d96e8a49ec2433c050targetblank2bba159a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/jdhenaos/invitro/tree/2bba1591ff879590b7dc34d96e8a49ec2433c050" target="_blank">2bba159</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Untracked files:
    Untracked:  analysis/at1_at2.Rmd
    Untracked:  code/cell_cycle.r
    Untracked:  code/common_genes.r
    Untracked:  code/deg_plots.r
    Untracked:  code/invivo.r
    Untracked:  code/single_cell.r
    Untracked:  code/unique_genes.r
    Untracked:  data/220427_ATII_embedding.rds
    Untracked:  data/220427_EC_embedding.rds
    Untracked:  data/220427_MFB_embedding.rds
    Untracked:  data/Count.matrix.xls
    Untracked:  data/G1S.txt
    Untracked:  data/G2M.txt
    Untracked:  data/Hyperoxia_AT.xls
    Untracked:  data/Hyperoxia_EC.xls
    Untracked:  data/Hyperoxia_MFB.xls
    Untracked:  data/bulk_seurat_endothelial.rds
    Untracked:  data/bulk_seurat_epithelial.rds
    Untracked:  data/bulk_seurat_stroma.rds
    Untracked:  data/inVivo_hyperoxia.txt
    Untracked:  data/inVivo_hyperoxia.xlsx
    Untracked:  data/samplesheet.txt
    Untracked:  et(frame,nsets = 20, nintersects = 15,

Unstaged changes:
    Modified:   README.md
    Modified:   _workflowr.yml
    Modified:   analysis/_site.yml
    Modified:   data/README.md
    Modified:   invitro.Rproj

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/invivo.Rmd</code>) and HTML (<code>docs/invivo.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jdhenaos/invitro/blob/2bba1591ff879590b7dc34d96e8a49ec2433c050/analysis/invivo.Rmd" target="_blank">2bba159</a>
</td>
<td>
jdhenaos
</td>
<td>
2025-09-02
</td>
<td>
workflowr::wflow_publish(“analysis/invivo.Rmd”)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(biomaRt)
library(gprofiler2)
library(ggplot2)
library(RColorBrewer)
library(GO.db)</code></pre>
<pre><code>Loading required package: AnnotationDbi</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, aperm, append, as.data.frame, basename, cbind,
    colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,
    get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,
    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,
    Position, rank, rbind, Reduce, rownames, sapply, saveRDS, setdiff,
    table, tapply, union, unique, unsplit, which.max, which.min</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:utils&#39;:

    findMatches</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    expand.grid, I, unname</code></pre>
<pre><code></code></pre>
<pre class="r"><code>library(UpSetR)

set_base_url(&quot;https://biit.cs.ut.ee/gprofiler_archive3/e102_eg49_p15&quot;)

FCplotter = function(pathways, DEG){
  # data is the dataframe containing only significant genes for a specific cell type
  # pathway is a df of pathway ids/go ids that are statistically enriched from geneprofile
  to.plot &lt;- data.frame()
  for(i in 1:nrow(pathways)){
    res = gost(query = pathways[i,&quot;term_id&quot;], organism = &quot;mmusculus&quot;)
    gene_set = res$meta$genes_metadata$query$query_1$ensgs
    
    to.plot &lt;- rbind(to.plot,
                     data.frame(
                       Name=pathways[i,&quot;term_name&quot;],
                       mean_FC=c(mean(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &gt; 0]),
                                 mean(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &lt; 0])),
                       Type=c(&quot;up-regulated&quot;,&quot;down-regulated&quot;),
                       Set_size = c(length(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &gt; 0]),
                                    length(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &lt; 0])),
                       SD=c(sd(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &gt; 0]),
                            sd(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &lt; 0]))
                     ))
  }
  
  to.plot[is.nan(to.plot$mean_FC),&quot;mean_FC&quot;] &lt;- 0
  
  return(to.plot)
}
enrichment_profile = function(pathways, DEG, color_down, color_up){
  # data is the dataframe containing only significant genes for a specific cell type
  # pathway is a df of pathway ids/go ids that are statistically enriched from geneprofile
  to.plot &lt;- data.frame()
  for(i in 1:nrow(pathways)){
    res = gost(query = pathways[i,&quot;term_id&quot;], organism = &quot;mmusculus&quot;)
    gene_set = res$meta$genes_metadata$query$query_1$ensgs
    
    to.plot &lt;- rbind(to.plot,
                     data.frame(
                       Name=pathways[i,&quot;term_name&quot;],
                       mean_FC=c(mean(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &gt; 0]),
                                 mean(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &lt; 0])),
                       Type=c(&quot;up-regulated&quot;,&quot;down-regulated&quot;),
                       Set_size = c(length(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &gt; 0]),
                                    length(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &lt; 0])),
                       SD=c(sd(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &gt; 0]),
                            sd(DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange[DEG[DEG$ensembl_gene_id %in% gene_set,]$log2FoldChange &lt; 0]))
                     ))
  }
  
  to.plot[is.nan(to.plot$mean_FC),&quot;mean_FC&quot;] &lt;- 0
  
  p &lt;- ggplot(to.plot, aes(x = mean_FC, y = reorder(Name, abs(mean_FC)), fill = Type, label = Set_size)) + 
    scale_fill_manual(name = &quot;&quot;, values = c(&quot;down-regulated&quot; = color_down, &quot;up-regulated&quot; = color_up)) + 
    ylab(&quot;&quot;) + xlab(&quot;Mean logFC&quot;) + 
    theme(axis.line = element_blank(),
          panel.grid.major = element_blank(),
          panel.grid.minor = element_blank(),
          panel.background = element_blank()) + theme(text = element_text(size = 7)) + 
    theme(legend.position = &quot;none&quot;) +
    geom_bar(stat = &quot;identity&quot;) + geom_text(size = 2, position = position_stack(vjust = 0.5)) +
    geom_linerange(aes(xmin=ifelse(mean_FC &lt; 0,mean_FC - SD, mean_FC), xmax=ifelse(mean_FC &gt; 0,mean_FC + SD, mean_FC) ))
  
  return(p)
}

data.folder &lt;- &quot;/workspaces/invitro/invitro/data/&quot;

raw.AT &lt;- read.table(paste0(data.folder,&quot;Hyperoxia_AT.xls&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)
DEG.AT &lt;- raw.AT[which(raw.AT$padj &lt; 0.05),]
raw.EC &lt;- read.table(paste0(data.folder,&quot;Hyperoxia_EC.xls&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)
DEG.EC &lt;- raw.EC[which(raw.EC$padj &lt; 0.05),]
raw.MFB &lt;- read.table(paste0(data.folder,&quot;Hyperoxia_MFB.xls&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)
DEG.MFB &lt;- raw.MFB[which(raw.MFB$padj &lt; 0.05),]
raw.IVC &lt;- read.table(paste0(data.folder,&quot;inVivo_hyperoxia.txt&quot;),sep = &quot;\t&quot;,header = T,stringsAsFactors = F)
DEG.IVC &lt;- raw.IVC[which(raw.IVC$padj &lt; 0.05),]

AT.gost &lt;- gost(DEG.AT$ensembl_gene_id,organism = &quot;mmusculus&quot;,sources=c(&quot;GO:MF&quot;, &quot;GO:BP&quot;,&quot;GO:CC&quot;, &quot;KEGG&quot;))
AT.BP &lt;- AT.gost$result[AT.gost$result$source == &quot;GO:BP&quot;,c(&quot;term_id&quot;, &quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;query_size&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;)]
EC.gost &lt;- gost(DEG.EC$ensembl_gene_id,organism = &quot;mmusculus&quot;,sources=c(&quot;GO:MF&quot;, &quot;GO:BP&quot;,&quot;GO:CC&quot;, &quot;KEGG&quot;))
EC.BP &lt;- EC.gost$result[EC.gost$result$source == &quot;GO:BP&quot;,c(&quot;term_id&quot;, &quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;query_size&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;)]
MFB.gost &lt;- gost(DEG.MFB$ensembl_gene_id,organism = &quot;mmusculus&quot;,sources=c(&quot;GO:MF&quot;, &quot;GO:BP&quot;,&quot;GO:CC&quot;, &quot;KEGG&quot;))
MFB.BP &lt;- MFB.gost$result[MFB.gost$result$source == &quot;GO:BP&quot;,c(&quot;term_id&quot;, &quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;query_size&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;)]
IVC.gost &lt;- gost(DEG.IVC$ensembl_gene_id,organism = &quot;mmusculus&quot;,sources=c(&quot;GO:MF&quot;, &quot;GO:BP&quot;,&quot;GO:CC&quot;, &quot;KEGG&quot;))
IVC.BP &lt;- IVC.gost$result[IVC.gost$result$source == &quot;GO:BP&quot;,c(&quot;term_id&quot;, &quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;query_size&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;)]

AT.dev &lt;- AT.BP[grep(pattern = &quot;develop&quot;,x = AT.BP$term_name),]
EC.dev &lt;- EC.BP[grep(pattern = &quot;develop&quot;,x = EC.BP$term_name),]
MFB.dev &lt;- MFB.BP[grep(pattern = &quot;develop&quot;,x = MFB.BP$term_name),]
IVC.dev &lt;- IVC.BP[grep(pattern = &quot;develop&quot;,x = IVC.BP$term_name),]</code></pre>
<div id="plots" class="section level1">
<h1>Plots</h1>
<div id="upset-plot" class="section level2">
<h2>Upset plot</h2>
<pre class="r"><code>frame &lt;- fromList(list(IVC.dev$term_id,
                       EC.dev$term_id,
                       MFB.dev$term_id,
                       AT.dev$term_id))

colnames(frame) &lt;- c(paste0(&quot;in vivo Hyperoxia&quot;,&quot; (N = &quot;,nrow(IVC.dev),&quot;)&quot;),
                     paste0(&quot;EC Hyperoxia&quot;,&quot; (N = &quot;,nrow(EC.dev),&quot;)&quot;),
                     paste0(&quot;MFB Hyperoxia&quot;,&quot; (N = &quot;,nrow(MFB.dev),&quot;)&quot;),
                     paste0(&quot;AT1/2 Hyperoxia&quot;,&quot; (N = &quot;,nrow(AT.dev),&quot;)&quot;))

#pdf(file = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_upsetplot.pdf&quot;,width = 7/2.54,height = 6/2.54)
upset(frame,nsets = 20, nintersects = 15, 
      text.scale = 1, 
      point.size = 1,
      line.size = 0.3,
      order.by = c(&quot;freq&quot;),empty.intersections = NULL,
      queries = list(list(query = intersects, 
                          params = list(&quot;AT1/2 Hyperoxia (N = 9)&quot;,&quot;in vivo Hyperoxia (N = 17)&quot;), 
                          color = &#39;#44AA99&#39;, active = T),
                     list(query = intersects, 
                          params = list(&quot;EC Hyperoxia (N = 8)&quot;,&quot;in vivo Hyperoxia (N = 17)&quot;), 
                          color = &#39;#117733&#39;, active = T),
                     list(query = intersects, 
                          params = list(&quot;AT1/2 Hyperoxia (N = 9)&quot;,&quot;in vivo Hyperoxia (N = 17)&quot;,&quot;EC Hyperoxia (N = 8)&quot;), 
                          color =  &#39;#332288&#39;, active = T)),
      sets.bar.color = c(&quot;#ffeda0&quot;,&quot;#A79AD4&quot;,&quot;#DF758C&quot;,&quot;#ACD49F&quot;),
      sets = colnames(frame),keep.order = T)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#dev.off()</code></pre>
</div>
<div id="invivo-at12" class="section level2">
<h2>invivo + AT1/2</h2>
<pre class="r"><code>AT.dev$term_name[1] &lt;- &quot;animal organ\ndevelopment&quot;
IVC.dev$term_name[1] &lt;- &quot;animal organ\ndevelopment&quot;

enrichment_profile(pathways = AT.dev[which(AT.dev$term_id %in% IVC.dev$term_id &amp; !AT.dev$term_id %in% EC.dev$term_id),],DEG = DEG.AT,color_down = &quot;#ACD49F&quot;,color_up = &quot;#008F00&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_AT/240110_AT.pdf&quot;,plot = p,width = 7,height = 3, units = &quot;cm&quot;)
enrichment_profile(pathways = IVC.dev[which(IVC.dev$term_id %in% AT.dev$term_id &amp; !IVC.dev$term_id %in% EC.dev$term_id),],DEG = DEG.IVC,color_down = &quot;#ffeda0&quot;,color_up = &quot;#feb24c&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-3-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_AT/240110_invivo.pdf&quot;,plot = p,width = 7,height = 3, units = &quot;cm&quot;)</code></pre>
</div>
<div id="invivo-ec" class="section level2">
<h2>invivo + EC</h2>
<pre class="r"><code>EC.dev$term_name[c(1,7,8)] &lt;- c(&quot;circulatory system\ndevelopment&quot;,&quot;regulation of\ndevelopmental process&quot;,&quot;anatomical structure\ndevelopment&quot;)
IVC.dev$term_name[c(4,5,10)] &lt;- c(&quot;anatomical structure\ndevelopment&quot;,&quot;circulatory system\ndevelopment&quot;,&quot;regulation of\ndevelopmental process&quot;)

enrichment_profile(pathways = EC.dev[which(EC.dev$term_id %in% IVC.dev$term_id &amp; !EC.dev$term_id %in% AT.dev$term_id),],DEG = DEG.EC,color_down = &quot;#A79AD4&quot;,color_up = &quot;#7030A0&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_EC/240110_EC.pdf&quot;,plot = p,width = 7,height = 3, units = &quot;cm&quot;)
enrichment_profile(pathways = IVC.dev[which(IVC.dev$term_id %in% EC.dev$term_id &amp; !IVC.dev$term_id %in% AT.dev$term_id),],DEG = DEG.IVC,color_down = &quot;#ffeda0&quot;,color_up = &quot;#feb24c&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-4-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_EC/240110_invivo.pdf&quot;,plot = p,width = 7,height = 3, units = &quot;cm&quot;)</code></pre>
</div>
<div id="invivo-at12-ec" class="section level2">
<h2>invivo + AT1/2 + EC</h2>
<pre class="r"><code>AT.dev$term_name[c(6,8)] &lt;- c(&quot;multicellular organism\ndevelopment&quot;,&quot;system\ndevelopment&quot;)
EC.dev$term_name[c(5,6)] &lt;- c(&quot;multicellular organism\ndevelopment&quot;,&quot;system\ndevelopment&quot;)
IVC.dev$term_name[c(2,3)] &lt;- c(&quot;system\ndevelopment&quot;,&quot;multicellular organism\ndevelopment&quot;)

enrichment_profile(pathways = AT.dev[which(AT.dev$term_id %in% IVC.dev$term_id &amp; AT.dev$term_id %in% EC.dev$term_id),],DEG = DEG.AT,color_down = &quot;#ACD49F&quot;,color_up = &quot;#008F00&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_AT_EC/240110_AT.pdf&quot;,plot = p,width = 7,height = 2, units = &quot;cm&quot;)
enrichment_profile(pathways = EC.dev[which(EC.dev$term_id %in% IVC.dev$term_id &amp; EC.dev$term_id %in% AT.dev$term_id),],DEG = DEG.EC,color_down = &quot;#A79AD4&quot;,color_up = &quot;#7030A0&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-5-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_AT_EC/240110_EC.pdf&quot;,plot = p,width = 7,height = 2, units = &quot;cm&quot;)
enrichment_profile(pathways = IVC.dev[which(IVC.dev$term_id %in% AT.dev$term_id &amp; IVC.dev$term_id %in% EC.dev$term_id),],DEG = DEG.IVC,color_down = &quot;#ffeda0&quot;,color_up = &quot;#feb24c&quot;)</code></pre>
<p><img src="figure/invivo.Rmd/unnamed-chunk-5-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#ggsave(filename = &quot;~/Documents/230516_hyperoxia/240110_common_development_paths/240110_invivo_AT_EC/240110_invivo.pdf&quot;,plot = p,width = 7,height = 2, units = &quot;cm&quot;)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-pc-linux-gnu
Running under: Ubuntu 22.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Etc/UTC
tzcode source: system (glibc)

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] UpSetR_1.4.0         GO.db_3.20.0         AnnotationDbi_1.68.0
 [4] IRanges_2.40.1       S4Vectors_0.44.0     Biobase_2.66.0      
 [7] BiocGenerics_0.52.0  RColorBrewer_1.1-3   ggplot2_3.5.2       
[10] gprofiler2_0.2.3     biomaRt_2.62.1       workflowr_1.7.1     

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.1        viridisLite_0.4.2       dplyr_1.1.4            
 [4] farver_2.1.2            blob_1.2.4              bitops_1.0-9           
 [7] filelock_1.0.3          Biostrings_2.74.1       RCurl_1.98-1.14        
[10] fastmap_1.2.0           lazyeval_0.2.2          BiocFileCache_2.14.0   
[13] promises_1.3.3          digest_0.6.37           lifecycle_1.0.4        
[16] processx_3.8.6          KEGGREST_1.46.0         RSQLite_2.3.7          
[19] magrittr_2.0.3          compiler_4.4.0          rlang_1.1.6            
[22] sass_0.4.10             progress_1.2.3          tools_4.4.0            
[25] yaml_2.3.10             data.table_1.17.8       knitr_1.50             
[28] labeling_0.4.3          prettyunits_1.2.0       htmlwidgets_1.6.4      
[31] bit_4.0.5               curl_7.0.0              plyr_1.8.9             
[34] xml2_1.4.0              withr_3.0.2             purrr_1.1.0            
[37] grid_4.4.0              git2r_0.33.0            scales_1.4.0           
[40] cli_3.6.5               rmarkdown_2.29          crayon_1.5.3           
[43] generics_0.1.4          rstudioapi_0.17.1       httr_1.4.7             
[46] DBI_1.2.3               cachem_1.1.0            stringr_1.5.1          
[49] zlibbioc_1.52.0         XVector_0.46.0          vctrs_0.6.5            
[52] jsonlite_2.0.0          callr_3.7.6             hms_1.1.3              
[55] bit64_4.0.5             plotly_4.11.0           jquerylib_0.1.4        
[58] tidyr_1.3.1             glue_1.8.0              ps_1.9.1               
[61] stringi_1.8.7           gtable_0.3.6            later_1.4.4            
[64] GenomeInfoDb_1.42.3     UCSC.utils_1.2.0        tibble_3.3.0           
[67] pillar_1.11.0           rappdirs_0.3.3          htmltools_0.5.8.1      
[70] GenomeInfoDbData_1.2.13 R6_2.6.1                dbplyr_2.5.0           
[73] httr2_1.2.1             rprojroot_2.1.1         evaluate_1.0.4         
[76] png_0.1-8               memoise_2.0.1           httpuv_1.6.16          
[79] bslib_0.9.0             Rcpp_1.1.0              gridExtra_2.3          
[82] whisker_0.4.1           xfun_0.53               fs_1.6.6               
[85] getPass_0.2-4           pkgconfig_2.0.3        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
