This is the methodology of the thesis.

\section{Data gathering}
    \subsection{Mice data}
        \subsubsection{Transcriptomics}
    \subsection{Human data}
        \subsubsection{Transcriptomics}
        \subsubsection{Metabolomics}
    \subsection{Public data}
        \subsubsection{Multi-omics bulk data}
        \subsubsection{Neonatal single-cell transcriptomics}
        
\section{Preprocessing}
    \subsection{Normalization}
        \subsubsection{DESeq2}
        \subsubsection{Pareto scaling}
        \subsubsection{Size-effect}
    \subsection{Data imputation}
        \subsubsection{Random-forest}
        \subsubsection{knn-Imputation}
    \subsection{Batch-effect detection}
        \subsubsection{Principal component analysis (PCA)}
        \subsubsection{Hierarchical clustering}
        \subsubsection{K-BET}

\section{Differential expression analysis}
    \subsection{Limma}
    \subsection{DESeq2}
    
\section{Enrichment analysis}
    \subsection{Gene list functional enrichment analysis}

\section{Multi-omics factor analysis (MOFA)}

\lipsum[1]

\section{Clinical data correlation}
    \subsection{Linear regression}
    \subsection{Binomial regression}
    \subsection{Ordinal regression}
    \subsection{Multinomial logistic regression}
    \subsection{Dirichlet regression}
    
\section{Benchmarking of Lasso models dealing with missing values}
    \subsection{Knowledge guided multi-level network inference}
    \subsection{Two-steps based models}
        \subsubsection{Grouped adaptive Lasso (GALasso)}
        \subsubsection{Stacked adaptive Lasso (SALasso)}
    \subsection{Inverse covariance based methods}
        \subsubsection{Convexed conditioned Lasso (CoCoLasso)}
        \subsubsection{Lasso with high missing rate (HMLasso)}

\section{Adult data correlation}
    \subsection{Random forest}
        \subsubsection{Imbalanced random forest}
        \subsubsection{Nested cross-validation in random forest}
    \subsection{t-test, manova, log-reg}

%TODO === workflow interoperability